src/emulate.c

```diff
@@ -1116,9 +1205,22 @@ void ebreak_handler(riscv_t *rv)
 void ecall_handler(riscv_t *rv)
 {
     assert(rv);
-#if RV32_HAS(SYSTEM)
-    syscall_handler(rv);
+
+/*
+ * FIXME: MMU test suite rely on rv32emu syscall instead of its own syscall
```

Comment:
The comment is a bit misleading. Place it in somewhere appropriate.

Reply:
> The comment is a bit misleading. Place it in somewhere appropriate.

Dropped it and mentioned in Makefile instead.